libs/base/Language/Reflection/TTImp.idr

```diff
@@ -129,10 +174,15 @@ mutual
 
   public export
   data DataOpt : Type where
+       ||| Determing arguements
        SearchBy : List1 Name -> DataOpt -- determining arguments
+       ||| Don't generate search hints for constructors
        NoHints : DataOpt -- Don't generate search hints for constructors
```

Comment:
Maybe, you should delete the comments after each constructor, since you added exactly same text to the docstring